tests/src/Unit/Plugin/TwigPlugin/JumpstartUITwigTest.php

```diff
@@ -58,6 +63,22 @@ public function testGetFunctions() {
     $functs = $this->twiggery->getFunctions();
     $this->assertInstanceOf(\Twig_SimpleFunction::class, $functs[0]);
     $this->assertEquals('getUniqueId', $functs[0]->getName());
+
+    $filters = $this->twiggery->getFilters();
+    $this->assertInstanceOf(\Twig_SimpleFilter::class, $filters[0]);
+    $this->assertEquals('render_clean', $filters[0]->getName());
+  }
+
+  /**
+   * Run the render_clean filter.
+   */
+  public function testsCleanFilter() {
+    $markup = '<div><a><span><article><section>test</section></article></span></a>';
```

Comment:
Perhaps a few special characters in here would be good too.

Reply:
I believe using the `#markup` does thay for us. I tested a few characters out

Reply:
Putting special characters in here are not modified in any way. by placing the resulting string into the array with `#markup` twig and drupal will do the encoding at that point. As long as this filter is not appended with `|raw` or something, there should be no issues.